--- configure.ac.rest	2011-04-14 17:50:58.000000000 +0200
+++ configure.ac	2011-04-14 17:51:13.000000000 +0200
@@ -30,7 +30,7 @@
 PKG_CHECK_MODULES(NBTK, nbtk-gtk-1.2)
 PKG_CHECK_MODULES(LIBNOTIFY, libnotify)
 PKG_CHECK_MODULES(MPL, moblin-panel)
-PKG_CHECK_MODULES(REST, rest-0.6)
+PKG_CHECK_MODULES(REST, rest-0.7)
 
 AC_MSG_CHECKING([for mobile-broadband-provider-info])
 PKG_CHECK_EXISTS(mobile-broadband-provider-info,
